driver/k8s/logicalvolume_service.go

```diff
@@ -140,7 +167,7 @@ func (s *LogicalVolumeService) CreateVolume(ctx context.Context, node, dc, name
 		if err != nil {
 			return "", err
 		}
-		logger.Info("created LogicalVolume CRD", "name", name)
+		logger.Info("created LogicalVolume CR", "name", name, "sourceID", lv.Spec.Snapshot.SourceID)
```

Comment:
I think the sourceId can be skipped in the log message. The LogicalVolume will have that information.

Reply:
Sure, will update it :+1: